lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3376,6 +3399,29 @@ namespace {
                                 std::nullopt);
     }
 
+    bool isNormalCxxMemberMethod(const clang::CXXMethodDecl *method) {
```

Comment:
What is the motivation here? If the reason to exclude some constructs because they cannot be annotated, I think we should spell this out somehow. There is no definition for 'normal' c++ member functions. I think this function should be renamed to something like "canBeRefcountConventionAnnotated" or something like to express the higher level goal.

Reply:
Yeah, it isn't clear what "normal" means here, I think the function name should convey the meaning as Gábor says.